app/models/job.js

```diff
@@ -200,7 +200,27 @@ export default Model.extend(DurationCalculations, {
     }
   }),
 
-  isTrustySudoFalse: Ember.computed.equal('queue', 'builds.ec2'),
+  isTrustySudoFalse: Ember.computed('queue', function () {
+    if (this.get('queue') === 'builds.ec2') {
+      return true;
+    }
+  }),
```

Comment:
I think this change is unnecessary, as the single-line `equal` computed property expresses the same thing more concisely.